test/test_files/rel_group/basic.test

```diff
@@ -66,6 +66,9 @@ Binder exception: Table knows does not exist.
 Binder exception: Table knows_personA_personB does not exist.
 -STATEMENT ALTER TABLE likes_personA_personB RENAME TO likes_A_B;
 ---- ok
+
+-CASE X
+-SKIP
```

Comment:
Why skip this part?

Reply:
Alter on REL_TABLE is temporarily disabled in this PR.